app/src/newdocs/content/SDKs.mdx

```diff
@@ -19,7 +26,14 @@ import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 <ScrollableAnchor id="java-sdk"><div>
 
 ## Java SDK
---content--
+
+The Java client library is available on [Maven Central](https://search.maven.org/search?q=g:com.streamr%20AND%20a:client&core=gav) which means it can be easily imported in your Java or Scala project. A comprehensive documentation and the code are available on [GitHub](https://github.com/streamr-dev/streamr-client-java).
```

Comment:
Wonder if it works with any JVM-based language? Clojure etc.?

Reply:
Probably yes, but don't know for sure.